src/Catty/ViewController/BaseViewController/BaseCollectionViewController.m

```diff
@@ -33,7 +33,7 @@
 
 @interface BaseCollectionViewController ()
 @property (nonatomic, strong) LoadingView* loadingView;
-@property (nonatomic, strong) ScenePresenterViewController *scenePresenterViewController;
+@property (nonatomic, strong) StagePresenterViewController *scenePresenterViewController;
```

Comment:
again - "stagePresenterViewController" instead of "scenePresenterViewController"